### PR TITLE
[ci skip] Fixing all <x> based occurances to <x>-based

### PR DESCRIPTION
**Pull request by AdityaBhutani**
### Summary

Fixed all <x> based occurances to <x>-based

as Suggested in [42479](https://github.com/rails/rails/pull/42479)

### CODE REVIEW

File: actionview/test/ujs/public/vendor/jquery-2.2.0.js [hunk 1]
```diff
@@ -6433,7 +6433,7 @@ Tween.propHooks = {
 };
 
 // Support: IE9
-// Panic based approach to setting things on disconnected nodes
+// Panic-based approach to setting things on disconnected nodes
```
Comment (p8) [inline actionview/test/ujs/public/vendor/jquery-2.2.0.js:6436]: This is third party code. No need to update it.

Comment (AdityaBhutani) [reply actionview/test/ujs/public/vendor/jquery-2.2.0.js]: Oh got It. Updated.

File: guides/source/4_2_release_notes.md [hunk 2]
```diff
@@ -819,7 +819,7 @@ Please refer to the [Changelog][active-support] for detailed changes.
     `Numeric#from_now`.
     ([Commit](https://github.com/rails/rails/commit/f1eddea1e3f6faf93581c43651348f48b2b7d8bb))
 
-*   Removed deprecated string based terminators for `ActiveSupport::Callbacks`.
+*   Removed deprecated string-based terminators for `ActiveSupport::Callbacks`.
```
Comment (zzak) [inline guides/source/4_2_release_notes.md:822]: We do not need to update these old release notes files.

**Review by zzak — CHANGES_REQUESTED** [review]
Could you please squash your commits?

**Review by zzak — APPROVED** [review]

**Pull request merged**